codeLabs-summer2021/3D-Model-World
# Local Storage

# Description
Added the ability to save and load models from the Local Storage.
If there are any models in the Local Storage they are automatically loaded upon starting the app IF LOGGED IN. If not Logged in they will be loaded once logged in*

*This _should_ work without extra code as when the user logs in, the site is refreshed causing the initial loading of models to be called

# To Test
1. Start the dev environment 
2. Log in using Sketchfab if needed
3. Load at least two models in the map
4. Verify the Local Storage is populated with the model's information
5. Make sure when you delete the model it is also deleted from Local Storage
6. Refresh the page
7. Verify the models are loaded from Local Storage
8. *Currently there is no way to check if the models are loaded upon log in. _(might need to push to live for that check)_
